mentoring-app/src/app/courses-page/course/course-item/course-item.component.html

```diff
@@ -1,16 +1,19 @@
-<mat-card (click)=goToCourse(course.id)>
+<mat-card (click)=goToCourse(course.id)
+        [appHighlightByDate]='course.creationDate'
+        [ngClass]="{'top-rated': course?.topRated}">
     <div class="course">
         <div class="course_date">
             {{course.creationDate | date}}
```

Comment:
Date format differs from one provided in task (9 Nov, 2018)

Reply:
fixed